RestoreSources.props

```diff
@@ -0,0 +1,9 @@
+<Project ToolsVersion="15.0">
+  <PropertyGroup>
+    <RestoreSources Condition="'$(RestoreSources)' == ''">
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
+      https://api.nuget.org/v3/index.json
```

Comment:
Usually a good idea to put a `;` after each entry to ensure any concatenation elsewhere works as expected. It is better to have more then less in this case.

Reply:
Yup, sounds good...